recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+
+[Service]
+User=root
+EnvironmentFile=-/etc/default/weston
+ExecStartPre=/usr/bin/weston-waitforcard.sh
```

Comment:
@kuscsik, turns out I had misconfigured the path unit.

FWIW, these are the results of the different approaches:

# Fails with "no drm device found" sometimes
Requires=dev-dri-card0.device

# Fails with "no drm device found" sometimes
Requires=dev-dri-card0.device
After=dev-dri-card0.device

# Locks sometimes on reboot with "A start job is running for dev-dri-card0.device (#s / 1min 30s)"
BindsTo=dev-dri-card0.device

# Works, should not be done
Restart=on-failure
RestartSec=5

# Seems to work OK!
Requires=weston.path
----vv----weston.path----vv----
[Path]
PathExists=/dev/dri/card0
----^^-------------------^^----